migrating/checklists/ossm-migrating-premigration-checklists-assembly.adoc

```diff
@@ -0,0 +1,258 @@
+** [ ] Create a standalone `Kiali` resource as the replacement. If you haven't already done this as part of the previous items for Prometheus or Tracing, follow link:https://docs.redhat.com/en/documentation/red_hat_openshift_service_mesh/3.0.0tp1/html/observability/kiali-operator-provided-by-red-hat#ossm-install-kiali-operator_ossm-kiali-assembly[Kiali Operator provided by Red Hat] on how to install and configure Kiali as a standalone resource.
+
+== Checklist for migrating {SMProduct} 2 routing and gateways
+
+This section applies if you have not already moved to explicitly-managed routes and gateway injection.
+
+Remember that automatic route creation, also known as Istio OpenShift Routing (IOR), is a deprecated feature that is disabled by default for any `ServiceMeshControlPlane` resource that was created using {SMProduct} 2.5 and later. To move from {SMProduct} 2 to {SMProduct} 3, you need to migrate from IOR to explicitly-managed routes.
+
+* [ ] Migrate from Istio OpenShift Routing (IOR) to to explicitly-managed routes. link:https://docs.redhat.com/en/documentation/openshift_container_platform/4.17/html/service_mesh/service-mesh-2-x#ossm-route-migration[Service Mesh Route Migration]
+
+Since the {SMProduct} 3 controle plane does not manage gateways, you must manage them yourself. To do so, and move move from {SMProduct} 2 to {SMProduct} 3, you need to disable the default gateways, move away from the Service Mesh Control Plane declaration, and begin to use gateway injection.
+
+* [ ] Migrate to gateway injection link:https://docs.redhat.com/en/documentation/openshift_container_platform/4.17/html/service_mesh/service-mesh-2-x#ossm-gateway-migration[Service Mesh Gateway Migration] to migrate off of the default ingress/egress gateways.
+
+== Checklist for disabling Grafana in {SMProduct} 2
+
+Grafana is not supported in {SMProduct} 3, and must be disabled in your {SMProduct} 2 `ServiceMeshControlPlane`:
+
+  * [ ] Disable Grafana: `spec.addons.grafana.enabled=false`.
+
+== Check your resource files
+
+By the end of this checklist, you {SMProduct} 2 resource files may look like the following examples:
+
+=== `ServiceMeshControlPlane` resource
+
+[source, yaml]
+----
+apiVersion: maistra.io/v2
+kind: ServiceMeshControlPlane
+metadata:
+  name: basic
+  namespace: istio-system
+spec:
+  version: v2.6 <1>
+  security: <2>
+    manageNetworkPolicy: false
+  addons: #<3>
+    grafana:
+      enabled: false
+    kiali:
+      enabled: false
+    prometheus:
+      enabled: false
+  meshConfig: <4>
+    extensionProviders:
+      - name: prometheus
+        prometheus: {}
+      - name: otel
+        opentelemetry:
+          port: 4317
+          service: otel-collector.istio-system.svc.cluster.local
+  gateways: <5>
+    enabled: false
+    openshiftRoute:
+      enabled: false
+  mode: MultiTenant <6>
+  tracing: <3>
+    type: None
+----
+
+<1> Your `ServiceMeshControlPlane` has been updated to the latest {SMv2Version} version.
```

Comment:
Here we have to be also careful about 2.6 version to be automatically updated. If it's not possible, we should use "use the latest"